src/AdobeAnalyticsHelper.php

```diff
@@ -319,6 +319,15 @@ protected function renderDatalayerMarkup(array $build) {
       return [];
     }
 
+    $environment = \Drupal::config('adobe_analytics')->get('mode');
```

Comment:
`adobe_analytics.mode` always returns `NULL`, therefore `$environment` is not being set. Perhaps `adobe_analytics.report_suite.report_suites_mode` should be used here?

Reply:
`adobe_analytics.mode` needs to be set in the `settings.php` file as 
```
$config['adobe_analytics'] = [
     'mode' => 'prod',
 ];
```

Reply:
Thanks for the clarification @prash-ittb, can we please have this documented in `README.md` to avoid any confusion.

Reply:
Have added the info to README file.